CONTRIBUTING.md

```diff
@@ -15,56 +15,54 @@ that we follow.
 ## Working On Issues
 
-Once you have decided to contribute to Podman by working on an issue, check our
-backlog of [open issues](https://github.com/containers/podman/issues) looking
-for any that do not have an "In Progress" label attached to it.  Often issues
-will be assigned to someone, to be worked on at a later time.  If you have the
-time to work on the issue now add yourself as an assignee, and set the
-"In Progress" label if you’re a member of the “Containers” GitHub organization.
-If you can not set the label, just  add a quick comment in the issue asking that
-the “In Progress” label be set and a member will do so for you.
+Once you have decided to contribute to Podman by working on an issue, check our backlog of [open issues](https://github.com/containers/podman/issues) looking for any that are unassigned.
+If you want to work on a specific issue that is already assigned, but does not appear to be actively being worked on, please ping the assignee in the issue and ask if you can take over.
+If they do not respond after several days, you can ping a maintainer to have the issue reassigned.
+When working on an issue, please assign it to yourself.
+If you lack permissions to do so, you can ping the `@containers/podman-maintainers` group to have a maintainer set you as assignee.
 
 ## Contributing to Podman
 
-This section describes how to start a contribution to Podman. These instructions are geared towards using a Linux development machine, which is required for doing development on the Podman backend. Check out these instructions for building the Podman client on [MacOSX](./build_osx.md) or
-[Windows](./build_windows.md).
+This section describes how to make a contribution to Podman.
+These instructions are geared towards using a Linux development machine, which is required for doing development on the Podman backend.
+Development for the Windows and Mac clients can also be done on those operating systems.
+Check out these instructions for building the Podman client on [MacOSX](./build_osx.md) or [Windows](./build_windows.md).
 
 ### Prepare your environment
 
 Read the [install documentation to see how to install dependencies](https://podman.io/getting-started/installation#build-and-run-dependencies).
 
 The install documentation will illustrate the following steps:
-- install libs and tools
-- check installed versions
-- configure network
-- how to install Podman from sources
+- Installation of required libraries and tools
+- Installing Podman from source
+
+The minimum version of Golang required to build Podman is contained in [go.mod](https://github.com/containers/podman/blob/main/go.mod#L5).
+You will need to make sure your system's Go compiler is at least this version using the `go version` command.
 
 ### Fork and clone Podman
 
-First you need to fork this project on GitHub.
+First, you need to fork this project on GitHub.
 
 Be sure to have [defined your `$GOPATH` environment variable](https://github.com/golang/go/wiki/GOPATH).
```

Comment:
This paragraph here and below is outdated, with modern go modules this stuff is really not needed